pkg/devfile/parser/data/v2/attributes.go

```diff
@@ -17,32 +17,36 @@ func (d *DevfileV2) GetAttributes() (attributes.Attributes, error) {
 	}
 }
 
-// UpdateAttributes updates the devfile top level attributes
-func (d *DevfileV2) UpdateAttributes(attr attributes.Attributes) error {
+// UpdateAttributes updates the devfile top level attribute for the specific key, err out if key is absent
+func (d *DevfileV2) UpdateAttributes(key string, value interface{}) error {
+	var err error
```

Comment:
any reason define an err here?

Reply:
because we use it here https://github.com/devfile/library/pull/82/commits/28cfac00e0399b7fdf10700a3e9a26e3bd6d01c6#diff-ab03248fb97b24c81ae668b35d88bb6f87073ee6739cf4b6dd754d86fb685b32R30